Cannot find module for any node_modules

When I use npm to install a package, I always get the following error when I require it in:

{
  "errorMessage": "Cannot find module 'jwt-simple'",
  "errorType": "Error",
  "stackTrace": [
    "Function.Module._load (module.js:276:25)",
    "Module.require (module.js:353:17)",
    "require (internal/module.js:12:17)",
    "Object.<anonymous> (/var/task/handler.js:3:11)",
    "Module._compile (module.js:409:26)",
    "Object.Module._extensions..js (module.js:416:10)",
    "Module.load (module.js:343:32)",
    "Function.Module._load (module.js:300:12)",
    "Module.require (module.js:353:17)"
  ]
}

The node_modules folder, and the normal serverless files (handler.js, serverless.yml etc) are all at the top level.

Interestingly - if I go into the aws lambda console and download the code it's uploaded, change the zip slightly (upload and then remove a random file) and then re-upload the code, it run's fine.

I guess it's an issue in the way that serverless creates the zip file, I'm running on a windows machine.

--- a/lib/plugins/package/lib/zipService.js
+++ b/lib/plugins/package/lib/zipService.js
@@ -51,8 +51,8 @@ module.exports = {
       }
     });
 
-    const platformName = ['UNIX', 'DOS', 'win32'].indexOf(process.platform) !== -1 ? process.platform :
-      'UNIX';
+    const platformName = ['UNIX', 'DOS', 'win32'].indexOf(process.platform) !== -1 ? 
+      process.platform : 'UNIX';
 
     return zip.generateAsync({
       type: 'nodebuffer',

--- a/lib/plugins/package/lib/zipService.js
+++ b/lib/plugins/package/lib/zipService.js
@@ -51,7 +51,7 @@ module.exports = {
       }
     });
 
-    const platformName = ['UNIX', 'DOS'].indexOf(process.platform) !== -1 ? process.platform :
+    const platformName = ['UNIX', 'DOS', 'win32'].indexOf(process.platform) !== -1 ? process.platform :
       'UNIX';
 
     return zip.generateAsync({

--- a/lib/plugins/package/lib/zipService.js
+++ b/lib/plugins/package/lib/zipService.js
@@ -51,7 +51,7 @@ module.exports = {
       }
     });
 
-    const platformName = ['UNIX', 'DOS', 'win32'].indexOf(process.platform) !== -1 ? 
+    const platformName = ['UNIX', 'DOS', 'win32'].indexOf(process.platform) !== -1 ?
       process.platform : 'UNIX';
 
     return zip.generateAsync({

